src/Solver.cpp

```diff
@@ -385,7 +352,55 @@ void Solver::useExponentialInterpolation() {
  *        to compute the exponential in the transport equation
  */
 void Solver::useExponentialIntrinsic() {
-  _interpolate_exponential = false;
+  _exp_evaluator->useIntrinsic();
+}
+
+
+/** 
+ * @brief Initializes new PolarQuad object.
+ * @details Deletes memory old PolarQuad if one was previously allocated.
+ */
+void Solver::initializePolarQuadrature() {
+
+  FP_PRECISION azim_weight;
+
+  /* Create Tabuchi-Yamamoto polar quadrature if a
+   * PolarQuad was not assigned by the user */
+  if (_polar_quad == NULL)
+    _polar_quad = new TYPolarQuad();
```

Comment:
I don't think this will ever be called since `_polar_quad` is set to `TYPolarQuad` during initialization of the `Solver`.